### ISSUE TITLE
Null pointer exception check suppressed just by checking on null, not whether actually returning/throwing

### ISSUE THREAD

**Issue by foobar13372** [kalessil/phpinspectionsea#1587]
```
$foo = getObjectFromSomewhereThatIsPossiblyNull();
if (null === $foo) {}
$foo->bar();
```
The pure existance of the `if` already suppresses the null pointer exception. It should be checked whether the condition actually has any effect (like return or throw).

**Comment by kalessil** [comment]
Thank you for reporting @foobar13372. You are right, and the issues were reported already, 
but the inspection is not taking care of workflow yet. I can not predict if this ever happens, 
but I'd advice using additional analyzers with more types controlling focus.
